src/device_update_import/mod.rs

```diff
@@ -0,0 +1,331 @@
+mod blob_uploader;
+use anyhow::{Context, Result};
+use azure_identity::{ClientSecretCredential, TokenCredentialOptions};
+use azure_iot_deviceupdate::DeviceUpdateClient;
+use blob_uploader::BlobUploader;
+use log::{debug, info};
+use serde::Serialize;
+use serde_json::json;
+use sha2::Digest;
+use std::collections::HashMap;
+use std::env;
+use time::format_description::well_known::Rfc3339;
+
+// See https://docs.microsoft.com/en-us/azure/iot-hub-device-update/device-update-limits
+const MAX_DEVICE_UPDATE_SIZE: u64 = 2000000000; // 2GB, may also actually be 2^32 - 1?
+const MANIFEST_VERSION: &str = "5.0";
+
+#[derive(Serialize)]
+struct UpdateId {
+    provider: String,
+    name: String,
+    version: String,
+}
+
+#[derive(Serialize)]
+#[serde(rename_all = "camelCase")]
+struct UserConsentHandlerProperties {
+    installed_criteria: String,
+}
+
+#[derive(Serialize)]
+#[serde(rename_all = "camelCase")]
+#[allow(non_snake_case)]
+struct SWUpdateHandlerProperties {
+    installed_criteria: String,
+    swuFileName: String,
+    arguments: String,
+    scriptFileName: String,
+}
```

Comment:
why are swuFileName and scriptFileName in camel case?

Reply:
https://github.com/omnect/iot-hub-device-update/blob/350a551dd9d3f5639eddceb75ef5b10e834865fe/src/extensions/download_handlers/README.md?plain=1#L33

Reply:
hmmm, not really a reason or i do not understand it

Reply:
Shouldn't the `serde(rename_all = "camelCase")` line above not take care of serializing them to their proper name, though?

Reply:
might be....  since the code is just copied from the old repo and it wasn't a finding there I have to take care of in the next PR